jdi-light/src/main/java/com/epam/jdi/light/common/Property.java

```diff
@@ -3,7 +3,11 @@
 public enum Property {
 
     TIMEOUT_WAIT_ELEMENT("timeout.wait.element", "^[1-9][0-9]{1,2}$|^\\d$",
-            "Value of 'timeout.wait.element' must be an integer from 0 to 999.");
+            "Value of 'timeout.wait.element' must be an integer from 0 to 999."),
+    SCREENS_FOLDER("screens.folder", "^(?:[a-zA-Z]\\:|\\\\\\\\[\\w\\.]+\\\\[\\w.$]+)\\\\(?:[\\w]+\\\\)*\\w([\\w.])+$", "'screens.folder' must contain the correct folder path"),
```

Comment:
I'm not sure enouth about this regexp

Reply:
Try to use this one:
^([A-z]:[\\\/]|[\\\/]|[\w+\.])+([\w+\.]+[\\\/]|[\w+\.])+$
it should match both win and unix paths. Doesn't allow spaces, but allows dots and underscores.
This one also should match paths like:
- my/predefined/path
- /my/relative/path
- C:\my\win\absolute\path

Reply:
Done